docs/source/modules/damping_driver.rst

```diff
@@ -0,0 +1,139 @@
+damping_driver.f90
+======================
+
+Summary
+-------
+The ``damping driver`` module is called by the ``idealized_moist_phys`` module by setting ``do_damping`` to true. It controls the upper level momentum damping in Isca. It controls 4 functions:
+
+1. **Rayleigh friction** which acts at levels ``1 to kbot``. This function is located in ``damping_driver`` itself.
+2. A (orographic) **mountain gravity wave drag** module may be called. This is the ``mg_drag`` module.
+3. A (non orographic) **convective gravity wave drag** module may be called. This is the ``cg_drag`` module.
+4. A **time independent drag** may be called. This function is located in ``damping_driver`` itself.
+
+There is another module references called ``topo_drag`` however this is not available in Isca at present. 
+
+Namelist options
+----------------
+``trayfric`` - (for Rayleigh friction) damping time in seconds for rayleigh damping momentum in the top model layers, the number of which is specified by ``nlev_rayfric`` (non namelist parameter, automatically determined in code). If ``trayfric`` < 0 then time in days. Default 0.
+
+``do_rayleigh`` - On/Off switch for doing Rayleigh friction. Default False
+
+``sponge_pbottom`` - (for Rayleigh friction) used to calculate ``nlev_rayfric``, it specifies the bottom level where the Rayleigh friction starts. Default 50Pa.
```

Comment:
typo: 50 hPa?

Reply:
Actually no, tho I suspect it should be. The default in the code is 50, but should probably be changed to 5000 or 15000. (50 or 150 hPa)